frontmacs-completion.el

```diff
@@ -29,8 +29,10 @@
 ;; Make the default completion mechanism a fuzzy search. However, you
 ;; don't really want to use fuzzy matching on lists that have content
 ;; with a lot of spaces (like documents), so disable for swiper.
+;; Disable for counsel-ag since it's not narrowing results with fuzzy.
 (setq ivy-re-builders-alist
       '((swiper . ivy--regex-plus)
+        (counsel-ag . ivy--regex-plus)
```

Comment:
This has been annoying me so much! THANK YOU 👍 👍 🙏

Reply:
Ugh, yea same here. As stated in b88e612

> A bug in swiper is causing counsel-ag to not narrow any results using
fuzzy search. This will be fixed in an upcoming release of swiper.

The present version is `swiper@0.11.0` as of this writing. So whatever the next release is we can update the package and remove that line.